.github/workflows/cmake.yml

```diff
@@ -36,4 +36,13 @@ jobs:
       # Build your program with the given configuration
       run: make
 
+    - name: Test Nonlinear Feature
+      working-directory: ${{github.workspace}}/test
+      run: clang-12 -emit-llvm -O3 -fno-unroll-loops -fno-vectorize -o nonlinear_test.bc -c nonlinear_test.cpp && opt-12 -load ./build/src/libmapperPass.so -mapperPass nonlinear_test.bc
+    
+    - name: Test Idiv Feature
+      working-directory: ${{github.workspace}}/test
+      run: clang-12 -emit-llvm -O3 -fno-unroll-loops -fno-vectorize -o idiv_test.bc -c idiv_test.cpp && opt-12 -load ./build/src/libmapperPass.so -mapperPass idiv_test.bc
+
```

Comment:
@HobbitQia @cwz920716 are you aware any testing strategy that can **check** whether a string(e.g., 4 `div`) in certain generated file (e.g., `dfg.png`).

Bash commands also fine for me..